docs/modules/ROOT/pages/how-tos/crossplane/enable_plan_upgrade.adoc

```diff
@@ -0,0 +1,67 @@
+= Enable Plan Upgrades
+
+This page describes the steps which are necessary to enable plan upgrades for both MariaDB and Redis.
+
+== Concepts
+
+Service plans consist of two different parts.
+The plan size and its SLA.
+
+We currently have two SLA levels: `standard` and `premium`. 
+SLA changes can be configured separately and doesn't need any of the setup described below.
+
+We're currently supporting five different plan sizes:
+
+* `xsmall`
+* `small`
+* `medium`
+* `large`
+* `xlarge`
+
+You can look at the corresponding xref:app-catalog:ROOT:redis.adoc[Redis] or xref:app-catalog:ROOT:mariadb_galera.adoc[MariaDB] pages to see what these translate to.
+Plan size changes are only supported if the following setup has been done.
+Updates are only valid from smaller to bigger plan sizes.
+Shrinking a service instance isn't supported.
+
+
+[NOTE]
+====
+It's configurable which plan changes the Service Broker will allow. 
+If the Service Broker is configured to for example allow a plan change from `large` to `small`, it will happily try to change the plan.
+So will the Crossplane Composition, but it will fail.
+
+Please make sure to now allow invalid plan changes when configuring the Service Broker.
```

Comment:
```suggestion
Since there's no programmatic guards against invalid plan changes in place, users have to ensure they don't allow invalid plan changes when configuring the Service Broker.
```